GlobalOptions.py

```diff
@@ -2,45 +2,68 @@
 
 class GlobalOptionsBuilder(ABC):
     '''Abstract Component Class'''
+    _globalOptionsList:list=[]
```

Comment:
No need for this, as it DafaultGlobalOptions Class is initializing the object to be decorated.